src/Fixer/Operator/OperatorLinebreakFixer.php

```diff
@@ -126,6 +127,7 @@ public function isCandidate(Tokens $tokens)
     protected function applyFix(\SplFileInfo $file, Tokens $tokens)
     {
         $referenceAnalyzer = new ReferenceAnalyzer();
+        $gotoLabelAnalyzer = new GotoLabelAnalyzer();
```

Comment:
We do we new this up ever time, for every file?

Reply:
Yeah that is the pattern currently, we could also initiate in the constructor and keep as privates. Sadly not all analyzer are stateless though (some require tokens to be passed). When that is fixed we can apply this new setup to all fixers in one go.

Reply:
Right. Is there an issue tracking this refactoring task?

Reply:
Alternatively, we could make this stuff a static method, and the same for the reference analyzer. I see no reason to make it an instance method, since we can't mock it, and it's internal and final.

Reply:
would be nice, however there are other member/maintainers that would like to see these analyzers etc. be injected using DI, so that is why we kind of stuck between moving towards that and using static method (or traits etc.)

Reply:
But this class is marked as internal, so nobody else is allowed to use it.

Reply:
sure, but if we make its method static it kind of makes it useless to inject later on